config/schedule.rb

```diff
@@ -36,6 +36,13 @@
   rake "email:renew_reminder:first:send"
 end
 
+# This is the daily second renewal reminder mail service.
+# Will run once a day in the early morning hours and send email reminders about
+# registrations that will expire in X time.
+every :day, at: (ENV["SECOND_RENEWAL_EMAIL_REMINDER_DAILY_RUN_TIME"] || "3:05"), roles: [:db] do
```

Comment:
🤦‍♂ 🤦‍♂ 🤦‍♂ 🤦‍♂ 🤦‍♂ 🤦‍♂ 

```suggestion
every :day, at: (ENV["SECOND_RENEWAL_EMAIL_REMINDER_DAILY_RUN_TIME"] || "04:05"), roles: [:db] do
```

Spotted we've defaulted the BOXI export at 3am 😳

Reply:
I should have spotted it myself :D